app/src/main/java/com/fmpdroid/moneybox/MainActivity.java

```diff
@@ -0,0 +1,55 @@
+package com.fmpdroid.moneybox;
+
+import androidx.appcompat.app.ActionBarDrawerToggle;
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.appcompat.widget.Toolbar;
+import androidx.core.view.GravityCompat;
+import androidx.drawerlayout.widget.DrawerLayout;
+
+import android.content.Intent;
+import android.os.Bundle;
+
+public class MainActivity extends AppCompatActivity {
+
+    private DrawerLayout drawer;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+
+        if (getIntent().getBooleanExtra("isEmpty", true)) {
```

Comment:
Put the string "isEmpty" in a Constant class since your are using this more than once.
Furthermore, use the said class for String constants. Name the variable that is specific to its job if the purpose for it is the same as this.